src/store/model/rosman-melb.ts

```diff
@@ -0,0 +1,145 @@
+import {Store} from './store';
+
+export const RosmanMelb: Store = {
+	backoffStatusCodes: [403, 429],
+	currency: '$',
+	labels: {
+		inStock: {
+			container:
+				'#Availability > div > table > tbody > tr:nth-child(2) > td:nth-child(1)',
+			text: [
+				'1',
+				'2',
+				'3',
+				'4',
+				'5',
+				'6',
+				'7',
+				'8',
+				'9',
+				'10',
+				'11',
+				'12',
+				'13',
+				'14'
+			]
+		},
+		maxPrice: {
+			container: 'span.price.price--withTax.price--main',
+			euroFormat: false
+		},
+		outOfStock: {
+			container:
+				'#Availability > div > table > tbody > tr:nth-child(2) > td:nth-child(1)',
+			text: ['ETA', 'Call Us', '0']
```

Comment:
```suggestion
			text: ['ETA', 'Call Us']
```

0 is probably not great here case `10` would also be part of this.

Reply:
included with build d33c279